frontend/src/component/Sidebar/WorkspaceListItem.jsx

```diff
@@ -49,7 +56,12 @@ const WorkspaceListItem = props => {
             color: color(GLOBAL_primaryColor).light() ? '#333333' : '#fdfdfd'
           }}
         >
-          {props.label.substring(0, 2).toUpperCase()}
+          {isDropActive
+            ? isDropAllowed && isDropAllowedOnWorkspaceRoot
+              ? <i className='fa fa-arrow-circle-down' />
+              : <i className='fa fa-times-circle' />
+            : props.label.substring(0, 2).toUpperCase()
```

Comment:
sould be a function

Reply:
It is the only place where we need this
I have moved this ternary into a specific function `getIcon()` to be clearer